app/serializers/webhook/article_serializer.rb

```diff
@@ -0,0 +1,38 @@
+module Webhook
+  class ArticleSerializer
+    extend ApplicationHelper
+    extend SocialImageHelper
+    include FastJsonapi::ObjectSerializer
+
+    set_type :article
+    attributes :title, :description, :readable_publish_date, :cached_tag_list, :cached_tag_list_array,
+               :slug, :path, :url, :comments_count, :positive_reactions_count, :body_markdown
+
+    attribute :canonical_url, &:processed_canonical_url
+    attribute :body_html, &:processed_html
+    attribute :created_at do |a|
+      a.created_at.utc.iso8601
+    end
+    attribute :edited_at do |a|
+      a.edited_at&.utc&.iso8601
+    end
+    attribute :crossposted_at do |a|
+      a.crossposted_at&.utc&.iso8601
+    end
+    attribute :published_at do |a|
+      a.published_at&.utc&.iso8601
+    end
+    attribute :last_comment_at do |a|
+      a.last_comment_at&.utc&.iso8601
+    end
+    attribute :cover_image do |a|
+      cloud_cover_url(a.main_image)
+    end
+    attribute :social_image do |a|
+      article_social_image_url(a)
+    end
+    attribute :user do |a|
```

Comment:
`fast_jsonapi` `belongs_to` only includes a reference to another record (type and id) https://github.com/Netflix/fast_jsonapi/issues/242 
I decided to include the user data in the article payload for now, regarding that we may want to remove ids from the payload and to limit the number of the API calls. 
I also made the payload matching the data in https://github.com/thepracticaldev/dev.to/blob/master/app/views/api/v0/articles/show.json.jbuilder

Reply:
It'd be great if in the PR description you could provide a JSON example of how the whole payload looks like. It's probably easier for reviewers

Reply:
Still haven't got to it, but I'll do that.

Reply:
I transformed the hash into JSON and pretty printed it here: https://gist.github.com/rhymes/06ab4ac2c45407f9c2ad9262c15d4506 - i linked this in the issue.

I like the format! I'm not 100% sure about your choice of ID though. I agree that it should be partially sortable but I think it should be global, not "per type". This way by looking at a possible future event log of these payloads (be it a table or a log destination or something else) it would be easy to figure out the sequence.

Reply:
Thanks for the formatting! I was in a hurry, but still decided to drop a link in case anyone will review.